ansible/roles/data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,124 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
+    repo: 'https://github.com/Sunbird-Ed/ml-analytics-service.git'
+    dest: "/opt/sparkjobs/ml-analytics-service"
+    version: main
+
+- name: CREATE THE USER
+  user:
+    name: "{{ USER }}"
+    state: present
+    home: "/home/{{ USER }}"
+    shell: /bin/bash
+
+- name: Ensure the locale exists
+  locale_gen:
+    name: en_US.UTF-8
+    state: present
+
+- name: set as default locale
+  command: locale-gen en_US.UTF-8
+
+- name: ADD THE REPOSITORY
+  apt_repository:
+    repo: "{{ item }}"
+  loop:
+    - "deb http://archive.ubuntu.com/ubuntu/ bionic universe"
```

Comment:
you can't have version specific repos. what if tomorrow we upgrade to 20 which is focal?

Reply:
and why are we adding universe repo? all the packages below are available from the default ones, right?

Reply:
we are facing issue with apt repository for universe,
That's why we went with shell command but as you suggested to use apt-repo,
i went with this approach 
This is were i refered 
https://github.com/ansible/ansible/issues/48714